docs/smart_scripts.md

```diff
@@ -47,64 +49,66 @@
 [8]: #event_flags
 [9]: #action_type
 [10]: #target_type
-[11]: #comment
+[11]: #target_x
+[12]: #target_y
+[13]: #target_z
+[14]: #target_o
+[15]: #comment
 
 **Description of the fields**
 
 ### entryorguid
 
--   EntryOrGuid &gt; 0: entry of the creature / game object / etc.
+- EntryOrGuid > 0: `entry` of the creature / game object / etc.
+- EntryOrGuid < 0: `guid` of the creature / game object / etc.
 
-<!-- -->
+**Depends on source\_type.**
```

Comment:
Make this a bulletpoint under line 63

Reply:
I don't understand what you mean by this.

Reply:
This is the only one I have yet to correct, but I did not understand what you are asking of me.

Reply:
Bulletpoint = -

- this is a bulletpoint

Reply:
A ok, now I add it.